boa/src/syntax/lexer/mod.rs

```diff
@@ -103,13 +105,14 @@ impl error::Error for LexerError {
 /// A lexical analyzer for JavaScript source code
 #[derive(Debug)]
 pub struct Lexer<'a> {
-    // The list fo tokens generated so far
+    /// The list of tokens generated so far.\
+    /// This field is public so you can use them once lexing has finished.
```

Comment:
I think we should use the more idiomatic rust documentation.
```
/// short summery
///
/// more details...
///
/// even more details
```
Instead of using the `\` trick.
```suggestion
    /// The list of tokens generated so far.
    ///
    /// This field is public so you can use them once lexing has finished.
```